### PR TITLE
Add deprecation messages to old ConfigurationManagement methods.

### PR DESCRIPTION
**Pull request by billfitzgerald0120**
Added log message to ConfigurationManagement methods.

https://www.pivotaltracker.com/story/show/143433335

### CODE REVIEW

**Comment by billfitzgerald0120** [comment]
@miq-bot add_label cleanup

**Comment by billfitzgerald0120** [comment]
@tinaafitz 
Please Review

**Comment by tinaafitz** [comment]
@billfitzgerald0120 Looks good.
@mkanoor Please review.

File: content/automate/ManageIQ/ConfigurationManagement/AnsibleTower/Operations/StateMachines/Job.class/__methods__/wait_for_completion.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ def initialize(handle)
   end
 
   def main
+    @handle.log("warn", "[DEPRECATION] This method will be deprecated. Please use AutomationManagement method instead.")
```
Comment (mkanoor) [inline content/automate/ManageIQ/ConfigurationManagement/AnsibleTower/Operations/StateMachines/Job.class/__methods__/wait_for_completion.rb]: @billfitzgerald0120 
Should the message say something like 
```
[DEPRECATION] This method will be deprecated. Please use similarly  named method from AutomationManagement namespace
```

**Comment by simaishi** [comment]
Fine backport details:

```
$ git log -1
commit 893f9bba6c98fd98d0059aa1f8ab0c8f1c1dd6ff
Author: Madhu Kanoor <mkanoor@redhat.com>
Date:   Tue Apr 11 17:03:22 2017 -0400

    Merge pull request #89 from billfitzgerald0120/deprecate_ConfigurationManagement
    
    Add deprecation messages to old ConfigurationManagement methods.
    (cherry picked from commit 85bc19a5cafa18002d191c32e5a7a8799b3c8ad1)
```

**Pull request merged**